chart/README.md

```diff
@@ -32,17 +32,18 @@ This repository is intended to store a helm chart to create a cluster of Nexus I
 ## Running
 
 1. Start your Kubernetes cluster if needed
-2. Open a console/terminal in the helm chart directory
+2. Open a console/terminal
 3. Switch to the correct context to use your cluster if needed (e.g. `kubectl config use-context my-context`)
-4. Install the helm chart dependencies via
-   `helm dependency update .`
+4. Add the helm chart repository via
+   `helm repo add sonatype https://sonatype.github.io/helm3-charts/`
 5. Install the helm chart via
-   `helm install --namespace <namespace> <name> <overrides> .`.
+   `helm install --namespace <namespace> <name> --dependency-update <overrides> sonatype/nexus-iq-server-ha --version <version>`.
```

Comment:
I would remove the period here because the "where" is part of the same sentence (and also to not make it look like part of the command)
```suggestion
   `helm install --namespace <namespace> <name> --dependency-update <overrides> sonatype/nexus-iq-server-ha --version <version>`
```